packages/itwinui-css/src/dialog/dialog.scss

```diff
@@ -74,6 +111,7 @@
   }
 
   .iui-dialog {
+    position: relative;
```

Comment:
is this needed?

Reply:
Yes it sets the default and pushes the dialog to the different corners based on that particular point on the screen.